src/fheroes2/gui/ui_button.cpp

```diff
@@ -425,25 +425,31 @@ namespace fheroes2
         Point offset;
 
         switch ( buttonTypes ) {
-        case Dialog::YES | Dialog::NO:
-            offset.x = area.x;
+        case Dialog::YES | Dialog::NO: {
+            const int32_t horizontalFreeSpace = area.width - AGG::GetICN( buttonYesIcnID, 0 ).width() - AGG::GetICN( buttonNoIcnID, 0 ).width();
+            const int32_t padding = horizontalFreeSpace / 4;
+
+            offset.x = area.x + padding;
             offset.y = area.y + area.height - AGG::GetICN( buttonYesIcnID, 0 ).height();
             createButton( offset.x, offset.y, buttonYesIcnID, 0, 1, Dialog::YES );
 
-            offset.x = area.x + area.width - AGG::GetICN( buttonNoIcnID, 0 ).width();
+            offset.x = area.x + area.width - AGG::GetICN( buttonNoIcnID, 0 ).width() - padding;
             offset.y = area.y + area.height - AGG::GetICN( buttonNoIcnID, 0 ).height();
             createButton( offset.x, offset.y, buttonNoIcnID, 0, 1, Dialog::NO );
-            break;
+        } break;
```

Comment:
Could you please move `break;` to the next line:
```suggestion
        }
        break;
```

Reply:
Hello @Districh-ru, thank you for the review.
This is what I had done initially but this was flagged by clang-format so I had to change it. 
https://github.com/ihhub/fheroes2/actions/runs/12497443035/job/34870080287
https://github.com/ihhub/fheroes2/pull/9385/commits/fb583c0aeea7dd9e1ec1c4405dab5125a56aeaa7

Reply:
Oh, you're right, @felix642!
But how about doing it another way? :)
```suggestion
            break;
        }
```